dev-docs/bidders/setupad.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: bidder
+title: Setupad
+description: Setupad bid adapter
+gvl_id: dev-docs/bidders/setupad.md
+tcfeu_supported: true
+usp_supported: true
+coppa_supported: false
+schain_supported: false
+dchain_supported: false
+safeframes_ok: false
+deals_supported: false
+floors_supported: false
+fpd_supported: false
+pbs_app_supported: false
+pbjs: true
+pbs: false
+biddercode: setupad
+media_types: banner
+prebid_member: true
+sidebarType: 1
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name | Scope | Description | Example | Type |
+|----------------+-------+-----------------------------------+-----------+---------|
+| `placement_id` | required | Placement ID, provided by setupad | `'12345'` | String |
+| `account_id` | optional | Account ID, provided by setupad | `'12345'` | String |
+
+### Additional options
+
+#### User Sync
+
+Add the following code to activate user sync. Setupad advises enabling user syncing through iFrames, as it significantly improves partners' user match rates and increases the Setupad bid rate and bid price. Be sure to call `pbjs.setConfig()` only once.
+
+```javascript
+pbjs.setConfig({
+  userSync: {
+    iframeEnabled: true,
+    filterSettings: {
+      iframe: {
+        bidders: '*', // '*' means all bidders
```

Comment:
It's not enough to allow syncing for your bidder?

Reply:
changes for 'setupad'. Is it ok now?

Reply:
Perfect :+1: If your bidder requires to allow user syncs for all bidders, this should highlighted